scripts/open_syllabus_project_parser.py

```diff
@@ -0,0 +1,22 @@
+{"ol_id":"/works/OL17946565W","Accounting":0,"Agriculture":0,"Anthropology":0,"Architecture":0,"Astronomy":0,"Atmospheric Sciences":0,"Basic Computer Skills":0,"Basic Skills":0,"Biology":0,"Business":0,"Career Skills":0,"Chemistry":0,"Chinese":0,"Classics":0,"Computer Science":0,"Construction":0,"Cosmetology":0,"Criminal Justice":0,"Criminology":0,"Culinary Arts":0,"Dance":0,"Dentistry":0,"Earth Sciences":0,"Economics":0,"Education":0,"Engineering":0,"Engineering Technician":0,"English Literature":9,"Film and Photography":0,"Fine Arts":0,"Fitness and Leisure":0,"French":0,"Geography":0,"German":0,"Health Technician":0,"Hebrew":0,"History":0,"Japanese":0,"Journalism":0,"Law":0,"Liberal Arts":0,"Library Science":0,"Linguistics":0,"Marketing":0,"Mathematics":0,"Mechanic / Repair Tech":0,"Media / Communications":0,"Medicine":0,"Military Science":0,"Music":0,"Natural Resource Management":0,"Nursing":0,"Nutrition":0,"Philosophy":0,"Physics":0,"Political Science":0,"Psychology":0,"Public Administration":0,"Public Safety":0,"Religion":0,"Sign Language":0,"Social Work":0,"Sociology":0,"Spanish":0,"Theatre Arts":0,"Theology":0,"Transportation":0,"Veterinary Medicine":0,"Women's Studies":0,"total":9}
+{"ol_id":"/works/OL2620037W","Accounting":0,"Agriculture":0,"Anthropology":0,"Architecture":0,"Astronomy":0,"Atmospheric Sciences":0,"Basic Computer Skills":0,"Basic Skills":0,"Biology":0,"Business":0,"Career Skills":0,"Chemistry":0,"Chinese":0,"Classics":0,"Computer Science":0,"Construction":0,"Cosmetology":0,"Criminal Justice":0,"Criminology":0,"Culinary Arts":0,"Dance":0,"Dentistry":0,"Earth Sciences":0,"Economics":0,"Education":0,"Engineering":0,"Engineering Technician":0,"English Literature":0,"Film and Photography":0,"Fine Arts":0,"Fitness and Leisure":0,"French":0,"Geography":0,"German":0,"Health Technician":0,"Hebrew":0,"History":0,"Japanese":0,"Journalism":0,"Law":0,"Liberal Arts":0,"Library Science":0,"Linguistics":0,"Marketing":0,"Mathematics":0,"Mechanic / Repair Tech":0,"Media / Communications":1,"Medicine":0,"Military Science":0,"Music":0,"Natural Resource Management":0,"Nursing":0,"Nutrition":0,"Philosophy":0,"Physics":0,"Political Science":0,"Psychology":0,"Public Administration":0,"Public Safety":0,"Religion":0,"Sign Language":0,"Social Work":0,"Sociology":0,"Spanish":0,"Theatre Arts":0,"Theology":0,"Transportation":0,"Veterinary Medicine":0,"Women's Studies":0,"total":1}
+'''  # noqa: E501
+
+from openlibrary.utils.open_syllabus_project import generate_osp_db
+
+input_directory = "/Users/rayberger/Downloads/drive-download-20230612T174927Z-001 (1)"
```

Comment:
Will discuss w Mek about best place to store this and potentially .db file.

Options
1. Put the .db file directly on prod/full-reindex servers. Con: It's kind of a magic file outside of our server/provisioning management systems
2. Put it in the archive behind creds, and download it.... at some point? And cache it in /tmp?

Reply:
I think #2 makes the most sense, if we have persistant data that we're processing and want to save, items seems like a good place for it. There's possibly a question about whether to restrict the item with s3 access but in this case I don't think that's necessary

Reply:
`2023_openlibrary_osp_counts`